lib/app/dispatcher.rb

```diff
@@ -27,13 +27,17 @@ class << self
       def start!
         running!
         Utility::Logger.info('Starting connector service...')
+        start_heartbeat_task!
```

Comment:
I don't see changes in spec files related to this change.

Reply:
Not yet. I just want to get feedback of this approach before I worked on specs.